forge-gui/res/cardsfolder/upcoming/landroval_horizon_witness.txt

```diff
@@ -0,0 +1,8 @@
+Name:Landroval, Horizon Witness
+ManaCost:4 W
+Types:Legendary Creature Bird Noble
+PT:3/4
+K:Flying
+T:Mode$ AttackersDeclared | Execute$ TrigPump | AttackedTarget$ Player | IsPresent$ Creature.attacking+YouCtrl | PresentCompare$ GE2 | NoResolvingCheck$ True | TriggerZones$ Battlefield | AttackingPlayer$ You | TriggerDescription$ Whenever two or more creatures you control attack a player, target attacking creature without flying gains flying until end of turn.
```

Comment:
It is `AttackersDeclaredOneTarget`
You probably need to add support for the amount

Reply:
Seems to work fine with the ispresent ?

Reply:
It's subtle but in multiplayer you should be able to get 2 triggers if you attack A + B with >=2 each only.
With your version 1 each would be enough.

@Hanmac 
your opinion?

Reply:
I'll remove this one for now